docs/download/repositories.md

```diff
@@ -36,15 +36,15 @@ Amazon cloud service containing ICGC data.
 
 | Property         | Value                                                                                                                                   |
 | :--------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
-| Name             | ICGC Storage Server (hosted at AWS)                                                                                                                 |
-| Contact          | <mailto:dcc-support@icgc.org>                                                                                                      |
+| Name             | ICGC Storage Server (hosted at AWS)                                                                                                     |
+| Contact          | [ICGC ARGO Contact page](https://platform.icgc-argo.org/contact)                                                                        |
 | Repository type  | Cloud                                                                                                      |
 | ICGC Portal Page | [Portal](https://dcc.icgc.org/repositories?filters=%7B%22file%22:%7B%22repoName%22:%7B%22is%22:%5B%22AWS%20-%20Virginia%22%5D%7D%7D%7D) |
 | Download Client  | [Tarball](/software/download/#score-client), [Docker](https://hub.docker.com/r/overture/score/) |
 | Repo Code        | `aws-virginia`                                                                                                                          |
 
 ### Obtaining Data Access
-Follow the procedure outlined at the [DACO page](https://icgc.org/daco).
+Follow the procedure outlined at the [DACO page][1].
```

Comment:
Same for here: 
To obtain access to Controlled Data, learn more about the [DACO application process, ](https://docs.icgc-argo.org/docs/data-access/daco/applying)or [start a DACO application here](https://daco.icgc-argo.org/).